client/utils/uploaderConfigs/getSharexConfiguration.ts

```diff
@@ -0,0 +1,32 @@
+class SharexConfiguration {
+  private Version = '15.0.0'
+  private Name: string
+  private DestinationType = 'ImageUploader, TextUploader, FileUploader'
+  private RequestMethod = 'POST'
+  private RequestURL: string
+  private Headers: object
+  private Body = 'MultipartFormData'
+  private FileFormName = 'file'
+  private URL: string
+  private ThumbnailURL: string
+  private DeletionURL = ''
+  private ErrorMessage = ''
+
+  constructor(name: string, baseURL: string, token: string) {
+    this.Name = name
+    this.RequestURL = `${baseURL}/api/files/upload/token`
+    this.URL = `${baseURL}/{json:code}`
+    this.ThumbnailURL = `${baseURL}/{json:code}`
+    this.Headers = {
+      'Content-Type': 'multipart/form-data',
```

Comment:
We HAVE TO remove this header, because it actually takes another parameter, which indicates the size of the content (it's omega dumb). So we let ShareX do it for us automatically.